conductr_cli/test/test_conduct_logs.py

```diff
@@ -33,22 +33,40 @@ def test_no_logs(self):
     def test_multiple_events(self):
         http_method = self.respond_with(text="""[
             {
-                "@cee": {
-                    "body": {
-                        "time": "Today 12:54:36",
-                        "host": "10.0.1.232",
-                        "log": "[WARN] [04/21/2015 12:54:30.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-22] Association with remote system has failed."
+                "@appname": "An App",
+                "@cee": {},
+                "sd": {
+                    "mdc": {
+                        "bundleId": "ddddaa3",
+                        "bundleName": "visualizer"
                     }
-                }
+                },
+                "@message": "[WARN] [04/21/2015 12:54:30.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-22] Association with remote system has failed.",
+                "@facility": "user",
+                "@hostname": "10.0.1.232",
+                "@msgid": "-",
+                "@procid": "-",
+                "@severity": "info",
+                "@timestamp": "2015-07-29T12:10:34+10:00",
+                "@sd": "[mdc source=\\\"conductr-event(akka://conductr)\\\" class=\\\"class akka.event.DummyClassForStringSources\\\"]"
             },
             {
-                "@cee": {
-                    "body": {
-                        "time": "Today 12:54:30",
-                        "host": "10.0.1.232",
-                        "log": "[WARN] [04/21/2015 12:54:36.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-26] Association with remote system has failed."
+                "@appname": "An App",
+                "@cee": {},
+                "sd": {
+                    "mdc": {
+                        "bundleId": "ddddaa3",
+                        "bundleName": "visualizer"
                     }
-                }
+                },
+                "@message": "[WARN] [04/21/2015 12:54:36.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-26] Association with remote system has failed.",
+                "@facility": "user",
+                "@hostname": "10.0.1.232",
+                "@msgid": "-",
+                "@procid": "-",
+                "@severity": "info",
+                "@timestamp": "2015-07-29T12:10:34+15:00",
+                "@sd": "[mdc source=\\\"conductr-event(akka://conductr)\\\" class=\\\"class akka.event.DummyClassForStringSources\\\"]"
             }
         ]""")
```

Comment:
Change the json we use in the test to reflect an actual example that might be sent from the logging endpoint.

Might need to update the json again to reflect the actual json sent by ElasticSearch.